doc/sphinx/language/core/variants.rst

```diff
@@ -6,11 +6,30 @@ Variants and the `match` construct
 Variants
 --------
 
+The :cmd:`Variant` command allows defining types by listing
+the :term:`inhabitants <inhabitant>` of the type. This includes enumerated types from
+programming languages, such as Booleans, characters or even the
+degenerate cases of the unit and empty types. Variant types more
+generally include enumerated types with arguments or even enumerated
+types with parametric arguments such as option types, sum types,
+etc. It also includes predicates or type families defined by cases
+such as the Boolean reflection or equality predicates. Observing the
+form of the :term:`inhabitants <inhabitant>` of a variant type is done by case analysis
+using the :cmd:`match` expression. Each case is described by what is
+called a :gdef:`constructor` of the type. For instance, Booleans have two
+constructors: :g:`true` and :g:`false`.
```

Comment:
Constructors apply equally to Variants, Inductives and CoInductives, so I think they should be described at the beginning of inductive.rst.

Probably should say constructors have names.

Reply:
Do you see a particular reason to define it at the beginning of inductive.rst rather than at the beginning of variants.rst (which comes first)?

Reply:
It will be fine in variants.rst.  (I looked at the order in which the changes appeared in github rather than in the actual manual.  I just got back to California Tuesday evening, so I'm a bit more addled than usual...)